circuits/cpp/src/aztec3/circuits/kernel/private/common.cpp

```diff
@@ -101,7 +95,7 @@ void common_validate_read_requests(DummyBuilder& builder,
         // but we use the leaf index as a placeholder to detect a transient read.
```

Comment:
```suggestion
        // but we use the leaf index as a placeholder to detect a 'pending note read'.
```
(And same for rest of this comment)